src/components/Forms/ApplyForm/ApplyForm.js

```diff
@@ -96,7 +96,7 @@ function ApplyForm() {
       </div>
       <div className="col-md-8">
         <div className="form-group">
-          <label htmlFor="lastNname" className="dark-text">
+          <label htmlFor="lastName" className="dark-text">
```

Comment:
Please add an `aria-label="last name"`

Reply:
I can add aria-label to these labels, do you want me to add it to all the labels on the page?  From my understanding of Aria specifications, adding aria-label is not needed here. You have used the correct semantic HTML with the appropriate attributes therefore by default these elements are accessible.  

I recently learned that although we use a variable type name to identify the input and label, when labeled correctly what is actually read by a screen reader is what is between the label tags.  <label .....attributes>Read this</label>.  Adding aria-label and aria-labelledby only needs to be added if for whatever reason there is no label element for you input element.

Reply:
This is a fantastic  resource for accessibility best practice - [Accessibility Oz](https://www.accessibilityoz.com/videos/)
I have it bookmarked and refer back to it all the time!